src/panels/GeoLocationPanel/AnchorPanels/EarthPanel/EarthPanel.tsx

```diff
@@ -79,7 +82,11 @@ export function EarthPanel({ currentAnchor }: Props) {
 
       setPlaces(uniquePlaces);
     } catch (error) {
-      console.error('Error fetching data:', error);
+      if (isReactRenderable(error)) {
+        showNotification('Error fetching data', error, LogLevel.Error);
```

Comment:
Same comment here as above.  This check should read "if we got an error" rather than "the error is a react node", and the code should be updated accordingly

Do you see the difference?